### PR TITLE
prometheus service annotations for elastisearch-exporter

### PR DESCRIPTION
**Pull request by Stelminator**
Adds annotations so that Prometheus running within Kubernetes will pick up the exposed metrics.

### CODE REVIEW

**Comment by Stelminator** [comment]
/assign @sameersbn

New to this repo, let me know what I can do to allow this to be merged, or whatever changes would be preferable.

File: stable/elasticsearch-exporter/templates/service.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ metadata:
     app: {{ template "elasticsearch-exporter.name" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  annotations:
```
Comment (unguiculus) [inline stable/elasticsearch-exporter/templates/service.yaml]: I'd suggest you rather make annotations configurable in general. If you use the prometheus-operator you don't need them.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR.

File: stable/elasticsearch-exporter/templates/service.yaml (hunk 1 above)
Comment (Stelminator) [reply stable/elasticsearch-exporter/templates/service.yaml]: updated, copied a pattern found in other charts.

**Pull request merged**